postmap: add page (#9600)
tldr-pages/tldr: pages/common/postmap.md

# postmap

> Create or query one or more Postfix lookup tables, or update an existing one.
> More information: <https://manned.org/postmap>.

> - Search specified maps for the key and remove one entry per map:

Reviewer: ```suggestion
- Search specific maps for a key and remove one entry per map (output stream: `stdout`):
```

```suggestion
- Search specific maps for a key and remove one entry per map (output stream: `stderr`):
```

Optional fix: `specific` may sound better.
Optional fix: add the correct output stream name explicitly.

> `postmap -d {{key}} {{input_file}}`

Reviewer 2: We're currently in the process of mandating that all paths start with `path/to/`.

```suggestion
`postmap -d {{key}} {{path/to/file}}`
```

- Enable message header query mode:

> `postmap -h {{input_file}}`

Reviewer 2: ```suggestion
`postmap -h {{path/to/file}}`
```

- Search specified maps for the key and write the first value found to the standard output stream:

> `postmap -q {{key}} {{input_file}}`

Reviewer 2: ```suggestion
`postmap -q {{key}} {{path/to/file}}`
```

- Retrieve all database elements, and write one line of key value output for each element:

> `postmap -s {{input_file}}`

Reviewer 2: ```suggestion
`postmap -s {{path/to/file}}`
```
